src/cli/commands/licenses.js

```diff
@@ -101,65 +99,4 @@ export const {run, setFlags, examples} = buildSubCommands('licenses', {
   async list(config: Config, reporter: Reporter, flags: Object, args: Array<string>): Promise<void> {
     await list(config, reporter, flags, args);
   },
-
-  async generateDisclaimer(config: Config, reporter: Reporter, flags: Object, args: Array<string>): Promise<void> {
```

Comment:
It is - it's an entry inside the object that is passed in parameter to `buildSubCommands`. It is run when you call `yarn licenses generate-disclaimer`.